main.go

```diff
@@ -139,6 +147,14 @@ func runRecordings(conf *Config, camera *lepton3.Lepton3) error {
 	maxFrames := conf.MaxSecs * framesHz
 	numFrames := 0
 	lastFrame := 0
+
+	recordingLed := gpioreg.ByName(conf.LEDs.Recording)
```

Comment:
Maybe return an error if this is `nil` to avoid a panic on the next line if the pin wasn't found.